dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/metadata/objectbundle/validation/SecurityCheck.java

```diff
@@ -138,6 +139,17 @@ private <T extends IdentifiableObject> void runValidationCheck(
         }
       }
 
+      if (object instanceof UserRole) {
```

Comment:
we could use the `instanceof` pattern match and remove the redundant cast e.g.
`object instanceof UserRole ur`

Reply:
This PR needs to be backported to Java 11 branches, so for convenience, it's better to refactor this later.